### PR TITLE
Update footer.html

### PR DESCRIPTION
**Pull request by starlord1311**
Made the changes

### CODE REVIEW

File: _includes/footer.html [hunk 1]
```diff
@@ -7,7 +7,9 @@
                     <div class="6u">
                         <h3><a href="{{'/events' | prepend: site.base_url }}">Upcoming Events</a></h3>
                         <ul class="alt">
-                            Coming Soon!
+                            Upcoming Events-
```
Comment (grubdragon) [inline _includes/footer.html]: Remove this line, "Upcoming Events" has already been written in line 8

File: _includes/footer.html [hunk 2]
```diff
@@ -7,7 +7,8 @@
                     <div class="6u">
                         <h3><a href="{{'/events' | prepend: site.base_url }}">Upcoming Events</a></h3>
                         <ul class="alt">
-                            Coming Soon!
+                            Introduction to competitive coding- CC-101 Date-Oct 4 2017 Time - 9 PM
```
Comment (grubdragon) [inline _includes/footer.html]: You haven't followed the template given in the comment below
<li><a href="#">Nascetur nunc varius</a></li>
"Nascetur nunc varius" is the event name

Comment (starlord1311) [reply _includes/footer.html]: I didn't get you

Comment (grubdragon) [reply _includes/footer.html]: `<ul>` indicates an unordered list. You have to add a list item with `<li>` tag. The list item template is given as `<li><a href="#">Nascetur nunc varius</a></li>`. Now figure out what you have to do!

File: _includes/footer.html [hunk 3]
```diff
@@ -7,7 +7,7 @@
                     <div class="6u">
                         <h3><a href="{{'/events' | prepend: site.base_url }}">Upcoming Events</a></h3>
                         <ul class="alt">
-                            Introduction to competitive coding- CC-101 Date-Oct 4 2017 Time - 9 PM 
+                            <!-- <li><a href="#">Introduction to competitive coding- CC-101 Date-Oct 4 2017 Time - 9 PM</a></li> -->
```
Comment (grubdragon) [inline _includes/footer.html]: This was almost correct, except for the fact that  are opening and closing tags of a comment in HTML
Remove these two from the line 10 and you are done

**Pull request closed without merging**